dali/test/python/test_operator_multipaste.py

```diff
@@ -0,0 +1,218 @@
+            kwargs["in_anchors"] = in_anchors
+
+        if not out_anchor_top_left:
+            kwargs["out_anchors"] = out_anchors
+
+        pasted = fn.multi_paste(resized, **kwargs)
+        pipe.set_outputs(pasted, resized)
+    return pipe, in_idx_l, in_anchors_l, shapes_l, out_anchors_l
+
+
+def manual_verify(batch_size, inp, output, in_idx_l, in_anchors_l, shapes_l, out_anchors_l, out_size_l):
+    for i in range(batch_size):
+        out = output.at(i)
+        out_size = out_size_l[i]
+        assert out.shape == out_size
+        ref = np.zeros(out.shape)
+        for j, idx in enumerate(in_idx_l[i]):
+            roi_start = in_anchors_l[i][j]
+            roi_end = roi_start + shapes_l[i][j]
+            out_start = out_anchors_l[i][j]
+            out_end = out_start + shapes_l[i][j]
+            ref[out_start[0]:out_end[0], out_start[1]:out_end[1]] = inp.at(idx)[roi_start[0]:roi_end[0],
+                                                                    roi_start[1]:roi_end[1]]
+        assert np.array_equal(out, ref)
+
+
+def show_images(batch_size, image_batch):
+    import matplotlib.gridspec as gridspec
+    import matplotlib.pyplot as plt
+    columns = 4
+    rows = (batch_size + 1) // (columns)
+    fig = plt.figure(figsize=(32, (32 // columns) * rows))
+    gs = gridspec.GridSpec(rows, columns)
+    for j in range(rows * columns):
+        plt.subplot(gs[j])
+        plt.axis("off")
+        plt.imshow(image_batch.at(j))
+    plt.show()
+
+
+def run_vs_python(bs, pastes, in_size, out_size, even_paste_count, no_intersections, full_input, in_anchor_top_left,
+                  out_anchor_top_left):
+    pipe, in_idx_l, in_anchors_l, shapes_l, out_anchors_l = get_pipeline(
+        batch_size=bs,
+        in_size=in_size,
+        out_size=out_size,
+        even_paste_count=even_paste_count,
+        k=pastes,
+        d_type=types.DALIDataType.UINT8,
+        no_intersections=no_intersections,
+        full_input=full_input,
+        in_anchor_top_left=in_anchor_top_left,
+        out_anchor_top_left=out_anchor_top_left
+    )
+    pipe.build()
+    print('build')
+    result, input = pipe.run()
+    if SHOW_IMAGES:
+        show_images(bs, result)
+    manual_verify(bs, input, result, in_idx_l, in_anchors_l, shapes_l, out_anchors_l, [out_size + (3,)] * bs)
+
+
+def test_simple():
+    tests = [
+        [4, 2, (128, 256), (128, 128), False, False, False, False, False],
+        [4, 2, (256, 128), (128, 128), False, True, False, False, False],
+        [4, 2, (128, 128), (256, 128), True, False, False, False, False],
+        [4, 2, (128, 128), (128, 256), True, True, False, False, False],
+
+        [4, 2, (64, 64), (128, 128), False, False, True, False, False],
+        [4, 2, (64, 64), (128, 128), False, False, False, True, False],
+        [4, 2, (64, 64), (128, 128), False, False, False, False, True]
+    ]
+    for t in tests:
+        print(f"Running with params: {t}")
+        run_vs_python(*t)
```

Comment:
```suggestion
        yield check_operator_multipaste, *t
```
Then you don't need the print (we use nosetests to run the test cases).
```
nosetests --verbose dali/test/python/test_operator_multipaste.py
```